core/dbt/graph/selector.py

```diff
@@ -329,16 +338,41 @@ def select_childrens_parents(self, selected: Set[str]) -> Set[str]:
         ancestors_for = self.select_children(selected) | selected
         return self.select_parents(ancestors_for) | ancestors_for
 
-    def select_children(self, selected: Set[str]) -> Set[str]:
+    def descendants(self, node, max_depth: int = None):
```

Comment:
The syntax for max_depth here should be `Optional[int]`, since it can be None. Also, please annotate the return type. I think this is accurate:

```suggestion
    def descendants(self, node, max_depth: Optional[int] = None) -> Set[str]:
```